README.md

```diff
@@ -1,10 +1,11 @@
-**Addons** is a repository of bleeding edge contributions that conform to
+**Addons** is a repository of contributions that conform to
 well-established API patterns, but implement new functionality
 not available in core TensorFlow. TensorFlow natively supports
 a larger number of operators, layers, metrics, losses, and optimizers.
 However, in a fast moving field like ML, there are many interesting new
 developments that cannot be integrated into core TensorFlow
-(because they are experimental, or their significance is not yet clear).
+(because their significance is not yet clear, or it is mostly used by a smaller
```

Comment:
"significance" seems so judgmental-- "scope"? Broad applicability? Just a suggestion-- fine as is, too.